### DISCUSSION TITLE
Support for non PDF formats

### DISCUSSION THREAD

**Discussion by mpflanzer** [papermerge/papermerge-core#137] [General]
Hi all,

That the support for non PDF files was dropped is a real pity. Makes it basically useless in my use case where I want to just quickly take pictures of invoices etc. with the phone and drop them somewhere for archiving, searching, etc.

What was the reason that the support was removed for now? Is there a timeline for when it will be re-enabled?

Also, almost everywhere in the docs support for JPG, PNG, TIFF is still advertised. Took quite a while until I spotted in the manual that the support was temporarily removed. Might make sense to put it more prominent on the main README to save people some frustration. Because the GUI is also not giving any message about why everything other than PDF files are still added but don't have a preview etc.

Cheers,
Moritz

**Comment by ciur** [comment]
Support for jpg, png and tiff will be re-added in version 2.3 which will be in this year (2023).
